tests/benchmarks/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 find_package(benchmark QUIET)
 
 # add_subdirectory(cluster)
```

Comment:
Are the rest of these benchmarks coming back later? I know there's an open issue for the cluster one.

Reply:
Yes, I plan to make a pass over them later. Some of them seem outdated, though. So probably not all of them will come back.